### PR TITLE
Tags

### PR DESCRIPTION
**Pull request by dafimor**
@refaelos pls review, thanks.

### CODE REVIEW

File: src/documents/university/articles/Tags_Dictionary.html.md [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+theme: 'platforms'
+collection: 'university_articles'
+module: 'articles'
+platform: 'university'
+---
+
+# Tags Dictionary
+
+## Tags Dictionary
+The games in the GROW network have all been tagged, and are defined by these tags. You can enter the God Mode in the [Dashboard](http://dashboard.soom.la/) and search for games with tags that are similar to your game, tags that are currently trending or just tags that interest you.
```
Comment (refaelos) [inline src/documents/university/articles/Tags_Dictionary.html.md]: `and are defined by these tags` - isn't that stating the obvious ?

The usage in God Mode is just one thing. You can write a whole paragraph about this alone. I would write one paragraph explaining the reasoning of using tags and what are they good for and another one about God Mode.

File: src/documents/university/articles/Tags_Dictionary.html.md [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+
+### Genre
+
+#### `Action`
+The action game emphasizes physical challenges, including hand–eye coordination and reaction-time. In an action game, the player typically controls the avatar/main character. The avatar must navigate a level, collecting objects, avoiding obstacles, and battling enemies with various attacks.
+<br/><br/>
+#### `Adventure`
+The action game emphasizes physical challenges, including hand–eye coordination and reaction-time. In an action game, the player typically controls the avatar/main character. The avatar must navigate a level, collecting objects, avoiding obstacles, and battling enemies with various attacks.
```
Comment (refaelos) [inline src/documents/university/articles/Tags_Dictionary.html.md]: Duplication of `Action`

**Comment by refaelos** [comment]
@dafimor some comments in side the code.

Also, add a one liner explaining each tag. 

Example: 

### Action
#### **In one line**: Physical challenges, including hand–eye coordination and reaction-time.

File: src/documents/university/articles/Tags_Dictionary.html.md [hunk 3]
```diff
@@ -0,0 +1,347 @@
...
+<br/><br/>
+#### `Arcade`
+Arcade games usually have very little puzzle-solving, complex thinking, or strategy skills needed; the focus is on reflexes and "twitch". The Arcade genre is usually used as an additional modifier to Action games.
+<br/><br/>
+#### `Board`
+Simulation of a traditional board game or party game where gameplay involves two or more players. A board game is a tabletop game that involves counters or pieces moved or placed on a pre-marked surface or "board", according to a set of rules. Games can be based on pure strategy, chance (e.g., rolling dice), or a mixture of the two, and usually have a goal that a player aims to achieve.
+<br/><br/>
+#### `Card`
+Simulation of a traditional board game or party game where gameplay involves two or more players. A board game is a tabletop game that involves counters or pieces moved or placed on a pre-marked surface or "board", according to a set of rules. Games can be based on pure strategy, chance (e.g., rolling dice), or a mixture of the two, and usually have a goal that a player aims to achieve.
```
Comment (refaelos) [inline src/documents/university/articles/Tags_Dictionary.html.md]: Duplication of `Board`

File: src/documents/university/articles/Tags_Dictionary.html.md [hunk 4]
```diff
@@ -0,0 +1,347 @@
...
+Any game using ideas, machines, weapons, or 'mechs similar to the FASA BattleTech series. This includes all 'Mech and "Giant Robot" games, including titles that are not explicitly from the FASA universe (Earthsiege, etc.).
+<br/><br>
+#### `Tiles`
+Computer implementation of a game played with tiles.
+<br/><br>
+#### `Candy`
+Any game that the gameplay evolves around Candy
+<br/><br>
+#### `Chess`
+Computer implementation of a game played with tiles.
```
Comment (refaelos) [inline src/documents/university/articles/Tags_Dictionary.html.md]: Same as `Tiles`

File: src/documents/university/articles/Tags_Dictionary.html.md [hunk 5]
```diff
@@ -0,0 +1,347 @@
...
+<br/><br>
+#### `Comics`
+Any comics related game, either characters from comics or comics look& feel...
+<br/><br>
+#### `Cyberpunk`
+Cyberpunk is a subgenre of science fiction in a future setting, noted for its focus on "high tech and low life" aspects of society. Cyberpunk plots often center on a conflict among artificial intelligences, and megacorporations, and tend to be set in a future Earth, rather than the far-future settings or galactic vistas.
+<br/><br>
+#### `Dark Sci-Fi`
+Cyberpunk is a subgenre of science fiction in a future setting, noted for its focus on "high tech and low life" aspects of society. Cyberpunk plots often center on a conflict among artificial intelligences, and megacorporations, and tend to be set in a future Earth, rather than the far-future settings or galactic vistas.
```
Comment (refaelos) [inline src/documents/university/articles/Tags_Dictionary.html.md]: Same as `Cyberpunk`

**Comment by dafimor** [comment]
@refaelos do we need the one-liner here (KB)?  I thought separately for Daniel for the Alt text in the dashboard

**Comment by refaelos** [comment]
@dafimor one liners can help here to understand the tag with a short description. It's always nicer to read one line then a whole description.

**Pull request closed without merging**